cli/src/semgrep/formatter/text.py

```diff
@@ -181,6 +185,91 @@ def _finding_to_line(
                 per_finding_max_lines_limit,
                 per_line_max_chars_limit,
                 show_separator,
+                False,
+            )
+
+    @staticmethod
+    def _match_to_lines(
+        location: out.Location,
+        content: str,
+        color_output: bool,
+        per_finding_max_lines_limit: Optional[int],
+        per_line_max_chars_limit: Optional[int],
+    ) -> Iterator[str]:
+        path = Path(location.path)
+        lines = get_lines(path, location.start.line, location.end.line)
+        yield from TextFormatter._format_lines(
+            path,
+            location.start.line,
+            location.start.col,
+            location.end.line,
+            location.end.col,
+            lines,
+            color_output,
+            per_finding_max_lines_limit,
+            per_line_max_chars_limit,
+            False,
+            True,
+        )
+
+    @staticmethod
+    def _call_trace_to_lines(
+        call_trace: out.CliMatchCallTrace,
+        color_output: bool,
+        per_finding_max_lines_limit: Optional[int],
+        per_line_max_chars_limit: Optional[int],
+    ) -> Iterator[str]:
+        trace = call_trace.value
+        if isinstance(trace, out.CliLoc):
```

Comment:
I don't need you to change this because I assume this predates your PR but what are these names

Reply:
"Location" versus "Call". Suppose I could have used "Match"